acre.yaml.example

```diff
@@ -20,17 +19,15 @@ landscape:
     credentials:
       # your credentials here
 
-  etcd:
-    backup:
-      type: <gcs/s3/abs>
-      region: europe-west1
-      credentials:
-        # your credentials here
+#  etcd: # optional
+#    backup:
+#      type: <gcs/s3/abs>
+#      region: (( iaas.region ))
+#      credentials: (( iaas.credentials ))
 
-  dns:
-    type: <google-clouddns/aws-route53/azure-dns>
-    credentials:
-      # your credentials here
+#  dns: # optional
```

Comment:
Why is the DNS block optional? What happens if I don't provide anything here?

Reply:
Also, where is the hosted zone information gone?

Reply:
Regarding the `hostedZoneID`: since the Gardener now also uses the DNS-controller-manager, the `hostedZoneID` is simply not needed anymore.

Reply:
If you leave out the `dns` block, it will be filled with default values from the `iaas` block: `dns.credentials` will be set to the same as `iaas.credentials` and `dns.type` will be derived from `iaas.type` (e.g. if `iaas.type` is `aws`, `dns.type` will be set to `aws-route53`). This works similarly for the `etcd` block (with the exception that the `resourceGroup` for Azure can`t be derived and needs to be specified).